streamlit_examples/utils/streamlit.py

```diff
@@ -43,3 +44,57 @@ def cache_file(file, type="pdf") -> str:
         with open(filepath, "wb") as f:
             f.write(file.getbuffer())
     return filepath
+
+def local_css(file_name):
+    with open(file_name) as f:
+        st.markdown(f'<style>{f.read()}</style>', unsafe_allow_html=True)
+
+
+def remote_css(url):
+    st.markdown(f'<link href="{url}" rel="stylesheet">', unsafe_allow_html=True)
+
+
+def icon(icon_name):
+    st.markdown(f'<i class="material-icons">{icon_name}</i>', unsafe_allow_html=True)
```

Comment:
@thucpn i guess you can move this HTML inside of the `st.components.v1.html` component? then we would get rid of the `markdown` components.

That way it should also be possible to reduce the height between the logo and the divider (the divider should be closer to the logo)
![image](https://github.com/marcusschiesser/streamlit-examples/assets/17126/b461ee3c-9fa0-4e50-baad-3135f567620e)